aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -468,7 +468,8 @@
       "permissions": [
         "rds:DeleteDBInstance",
         "rds:DescribeDBInstances"
-      ]
+      ],
+      "timeoutInMinutes": 2160
```

Comment:
Is parameter only used for documentation or it is applied logically in handlers?

Reply:
It is applied logically in handlers